terra_notebook_utils/cli/workflows.py

```diff
@@ -0,0 +1,55 @@
+import json
+import argparse
+from typing import Any, Dict
+
+from terra_notebook_utils import workflows
+from terra_notebook_utils.cli import Config, dispatch
+
+
+workflow_cli = dispatch.group("workflows", help=workflows.__doc__)
```

Comment:
Could a docstring be added?  `workflows.__doc__` is blank atm.

Reply:
will do